utils/ValidateReadmeTable.py

```diff
@@ -0,0 +1,62 @@
+import os
+import re
+import sys
+from typing import Final
+
+CHECK_ID_PATTERN: Final[str] = r"(PWR\d+|PWD\d+|RMK\d+)"
+CHECKS_DIR: Final[str] = "Checks/"
+README_FILE: Final[str] = "README.md"
+TABLE_MARKER: Final[str] = "|"
+
+
+def extractTableFromReadme(readmeFile: str) -> list[str]:
+    with open(readmeFile, "r") as file:
+        lines = file.readlines()
+
+    tableLines = [
+        line for line in lines if line.strip() and line.startswith(TABLE_MARKER)
+    ]
+
+    return tableLines
+
+
+def extractIdsFromTable(tableLines: list[str]) -> set[str]:
+    return {
+        match.group(1)
+        for line in tableLines
+        if (match := re.search(CHECK_ID_PATTERN, line))
+    }
+
+
+def extractIdsFromChecksDirectory(cheksDirectory: str) -> set[str]:
+    return {entry.name for entry in os.scandir(cheksDirectory) if entry.is_dir()}
+
+
+def main():
+    tableContents: Final[list[str]] = extractTableFromReadme(README_FILE)
+    tableIds: Final[set[str]] = extractIdsFromTable(tableContents)
+    checksDirIds: Final[set[str]] = extractIdsFromChecksDirectory(CHECKS_DIR)
+
+    missingFromTable: Final[set[str]] = checksDirIds - tableIds
+    extraInTable: Final[set[str]] = tableIds - checksDirIds
+
+    if missingFromTable:
+        print(
+            f"[Error] The following IDs are missing from the README table: "
+            f"{', '.join(missingFromTable)}"
+        )
+    if extraInTable:
+        print(
+            f"[Error] The following IDs are listed in the README table but do "
+            f"not exist in the checks directory: {', '.join(extraInTable)}"
+        )
```

Comment:
Pretty nice you add the diagnostics! :heart: 

But to be fair, I expect that the one seen this PR failing, knows which check is missing :rofl:

Reply:
> But to be fair, I expect that the one seen this PR failing, knows which check is missing 🤣

Let's hope so :grin: 

I'll leave it anyway if it's ok. Better to be explicit if the script fails, than to leave room for doubt.